grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -584,21 +584,22 @@ class InventoryItemController {
         }
 
         log.info("User chose to validate or there are errors")
-        def warehouseInstance = Location.get(session?.warehouse?.id)
-
+        Location warehouseInstance = Location.get(session?.warehouse?.id)
+        commandInstance.product = Product.load(commandInstance.product.id)
```

Comment:
why is this needed? Was the product in the command not getting initialized properly?